src/decode.rs

```diff
@@ -4845,9 +4847,10 @@ pub(crate) unsafe fn rav1d_decode_frame(c: &Rav1dContext, f: &mut Rav1dFrameData
             res = rav1d_decode_frame_main(c, f);
             let frame_hdr = &***f.frame_hdr.as_ref().unwrap();
             if res.is_ok() && frame_hdr.refresh_context != 0 && f.task_thread.update_set {
+                let out_cdf = f.out_cdf.cdf().unwrap();
```

Comment:
The `refresh_context != 0` and `.unwrap()` can be combined with an `if let Some`.

Reply:
`cdf_write()` now tries to lock, so we need the unwrap